clients/cloud/confluent-cli/README.md

```diff
@@ -13,10 +13,10 @@ To run this example, create a local file with configuration parameters to connec
 If this is a Confluent Cloud cluster, you must have:
 
 * Access to a [Confluent Cloud](https://www.confluent.io/confluent-cloud/?utm_source=github&utm_medium=demo&utm_campaign=ch.examples_type.community_content.clients-ccloud) cluster
-* Initialize a properties file at `$HOME/.ccloud/config` with configuration to your Confluent Cloud cluster:
+* Initialize a properties file at `$HOME/.confluent/config` with configuration to your Confluent Cloud cluster:
```

Comment:
This needs to be replaced as well, as was done for clojure.  And I don't think it will be called `$HOME/.confluent/config`, won't it be called `$HOME/.confluent/java.config`?